index.bs

```diff
@@ -2537,13 +2537,15 @@ Short Circuit</h5>
 	and potential members of a Council who are not renouncing their seat
```

Comment:
```suggestion
	and potential members of a Council from elected positions who are not renouncing their seat
```
This has effect of subsetting the short-circuit vote to the elected members of the AB + TAG, which I believe is correct, as the CEO and TimBL have likely already had the chance to contribute to the Team recommendation, and furthermore unburdens the appointed TAG members from feeling pressured to vote (or not) in alignment with the Team who appointed them.

Reply:
The problem statement (from #784) for this pull request is:
> @timbl was unintentionally left on the critical path of W3C Councils.

I think what you're trying to address here is a distinct question (or even two). I'd suggest opening a new issue/PR (or two) to discuss that separately.

As feedback on this: I think treating the CEO and Director emeritus as part of the Team, and therefore assuming their assent is implied in a Team recommendation is reasonable. However, that logic doesn't apply to appointed TAG members, and the rationale there is different, as you noted. I'm a lot less sure I agree with that part, and in any case, I'd suggest a separate discussion.

Reply:
I agree. I think once they are on teh TAG, they should be treated the same as every other member.

Reply:
@dwsinger wrote:
> I agree. I think once they are on the TAG, they should be treated the same as every other member.

I suppose now that TAG appointees cannot be re-appointed (they have to run for election explicitly to continue serving after their appointed term ends), there is less ongoing connection, potential pressure from, or sense of obligation to the Team appointing them. I'll make a minor edit accordingly at least to capture these suggestions before creating a new issue/PR as @frivoal suggests.